syndesis/tests/org.fusesource.ide.syndesis.extension.ui.tests.integration/src/main/java/org/fusesource/ide/syndesis/extensions/tests/integration/SyndesisExtensionProjectCreatorRunnableIT.java

```diff
@@ -279,4 +280,12 @@ public MavenExecutionResult call(IMavenExecutionContext context, IProgressMonito
 		}
 		assertThat(buildOK).isTrue();
 	}
+	
+	/* (non-Javadoc)
+	 * @see org.fusesource.ide.camel.tests.util.AbstractProjectCreatorRunnableIT#checkCorrectNatureEnabled(org.eclipse.core.resources.IProject)
+	 */
+	@Override
+	protected void checkCorrectNatureEnabled(IProject project) throws CoreException {
+		// we don't want to check for rider nature
```

Comment:
why?
Don't we want it when there is a Camel XML route defined?

Reply:
That would enable them to change camel versions as they like and I don't want that. Camel editor functionality is still there, thats enough.